app/containers/MessageBox/FilesActions.js

```diff
@@ -26,6 +27,10 @@ export default class FilesActions extends PureComponent {
 		this.options.push(I18n.t('Choose_from_library'));
 		this.LIBRARY_INDEX = 2;
 
+		// File
+		this.options.push(I18n.t('Choose_file'));
```

Comment:
I suggest to `this.options = [Cancel, TakePhoto...]` no need for all these `this.options.push`

Reply:
It's better readable with it1n